scripts/build.js

```diff
@@ -18,9 +19,26 @@ const FILES_TO_COPY = [
 
 shell.echo(`Start building...`);
 
-shell.mkdir(`${NPM_DIR}`);
-shell.rm(`-Rf`, `${NPM_DIR}/*`);
-shell.cp(`-Rf`, FILES_TO_COPY, `${NPM_DIR}`);
+shell.mkdir(`-p`, `${NPM_DIR}`);
+
+// Check whether build folder is empty. If so delete any contents. 
+const folderSize =  parseInt(shell.exec(`du -s ${NPM_DIR} | cut -f1`).stdout);
+const folderContents =  shell.exec(`ls -A ${NPM_DIR}`).stdout;
+if ((folderSize && folderSize > 0) || (folderContents && folderContents != '')) {
+  shell.echo(`Removing contents of build folder...`);
+  shell.rm(`-Rf`, `${NPM_DIR}/*`);
+}
```

Comment:
Perhaps mention that the folder already exists with contents should this `if` not happen?  That could help with someone debugging.

Reply:
My bad, comment is confusing.